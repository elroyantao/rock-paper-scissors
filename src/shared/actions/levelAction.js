export const changeLevel = (level) => ({
  type: 'CHANGE_LEVEL',
  level
})
