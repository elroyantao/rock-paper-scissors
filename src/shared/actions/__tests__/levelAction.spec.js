import * as actions from '../levelAction'

const snapshot = (action) => expect(action).toMatchSnapshot()

describe('levelAction', () => {
  it('changeLevel', () => {
    snapshot(actions.changeLevel('advanced'))
  })
})
