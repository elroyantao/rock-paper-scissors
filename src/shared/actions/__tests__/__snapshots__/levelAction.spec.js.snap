// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`levelAction changeLevel 1`] = `
Object {
  "level": "advanced",
  "type": "CHANGE_LEVEL",
}
`;
